### PR TITLE
docs(updater): update Auto Update quick setup guide

### PR DESCRIPTION
**Pull request by changkun**
1. Add zip target attention note for macOS
2. Add `electron-updater` quick setup code snippet example

See: https://github.com/electron-userland/electron-builder/issues/1367

### CODE REVIEW

**Comment by develar** [comment]
Maybe it is better to use notifications instead of dialog box (https://github.com/develar/onshape-desktop-shell/blob/master/src/AppUpdater.ts)?

**Comment by changkun** [comment]
@develar It is arguable to use messaging or dialog box.

According to my practice, one of the drawbacks for notification is to set up an ipcRenderer somewhere else if menu creates on ipcMain, which may lead coupling for afterward maintains. I would rarely use it. In this snippet, the complete update function was encapsulated, and the `updater` is not related to anyone else.

Please let me know more benefits for notification usage scenario if I am wrong.

**Comment by develar** [comment]
@changkun I don't like message box because it interrupts user work. Notifications are much more user-friendly. I don't want to promote message box.

Yeach, electron supports notifications only for renderer process and as result, example will be complicated.

Also, `autoUpdater.autoDownload = false` is promoted, but I think that for most users auto download is better.

Maybe we should add note about it like "this example for manual update via menu".

File: docs/Auto Update.md [hunk 1]
```diff
@@ -26,9 +26,13 @@ Simplified auto-update is not supported for Squirrel.Windows.
...
 ### Examples
 
+**Auto Update**
+
 A [complete example](https://github.com/iffy/electron-updater-example) showing how to use.
 
-The following code snippet give another example of how to implement few `electron-updater` events, which gives a optional update function.
+**Manual Update**
+
+The following code snippet gives another example, which illustrate an encapsulated menual update via menu.
```
Comment (develar) [inline docs/Auto Update.md]: menual -> `manual`

**Comment by changkun** [comment]
@develar You are definitely right about the interruption. I also hate it. For manual update, user should aware that they are updating software and will be interrupt in the furture, does it reasonable a little?

**Comment by develar** [comment]
> You are definitely right about the interruption. I also hate it. For manual update, user should aware that they are updating software and will be interrupt in the furture, does it reasonable a little?

For manual auto update message box is ok. But in general I don't want to promote such implementation. I am afraid that most users will stupidly and blindly copy example.

**Comment by develar** [comment]
Thanks! I am not happy to promote this example, but for now ok.

**Comment by changkun** [comment]
You're welcome. Perhaps you can remove the example from your subsequent commit :-)

**Pull request merged**